src/app/home/home.component.html

```diff
@@ -24,10 +24,10 @@
 <mat-menu #notificationMenu="matMenu" [overlapTrigger]="false">
   <span mat-menu-item *ngFor="let notification of notifications" (click)="readNotification(notification.doc)">
     <mat-divider></mat-divider>
-    <p [ngStyle]="{'color': 'green'}" *ngIf="notification.doc.status==='unread' && notification.doc.link">
+    <p class="menu-Item-notification" *ngIf="notification.doc.status==='unread' && notification.doc.link">
       <a href="{{notification.doc.link}}">{{notification.doc.message}} {{notification.doc.time | date: 'MMM d, yyyy'}}</a>
     </p>
-    <p [ngStyle]="{'color': 'green'}" *ngIf="notification.doc.status==='unread' && notification.doc.link ===''">
+    <p class="menu-Item-notification" *ngIf="notification.doc.status==='unread' && notification.doc.link ===''">

```

Comment:
Could you please check the naming structure ? class="menu-Item-notification" should be like menu-item-notification